### DISCUSSION TITLE
#06th Sprint - More UI Demo, GitHub pages and pancakes

### DISCUSSION THREAD

**Discussion by jas20202** [CUMGroup/AMOGUS#32] [Blog]
Heyyo AMOGUS friends!

Today we want to present you our progress this week so far. This sprint has more workload than the previous ones as we are now working more and more on the demo parts of our project.

But now we have some progress to show 😁

@AlexMi-Ha has worked on a demo for our stats view based on our previously determined [userstory](https://github.com/CUMGroup/AMOGUS/blob/main/documents/requirements/usecases/stats.usecase.md) and [this plots and stats schematics](https://github.com/CUMGroup/AMOGUS/blob/main/documents/gameplay/plots_statstracking.pdf). Below you can see a screenshot of the implementation in Angular which Alex worked on the past week:
![image](https://user-images.githubusercontent.com/99250573/203346801-2b05aa24-ba59-44bd-8e86-e95771c82e0b.png)

Furthermore the code from last weeks demo was restructured and is now in the main branch!

@WachtelHD has been working on our login and register pages (Screenshots coming soon) while @Annalytic-programming is working on our Authentication Service in the frontend side. 

Meanwhile @AlexMi-Ha and @jas20202 set up GitHub Pages for our project in order to have a pleasing and well structured user and developer documentation. The content for the documentation pages are yet to come but you can already visit the GitHub Pages of our project with the following link: https://cumgroup.github.io/AMOGUS/
We decided to use [MkDocs](https://www.mkdocs.org/) and the [material theme](https://squidfunk.github.io/mkdocs-material/) for MkDocs to create the content for our GitHub Pages.

Every team member, who hasn't used Angular yet, is now familiar with the basics of Angular and ready for some tasks.

And now, as promised in the title, pancakes: 🥞🥞🥞
![image](https://user-images.githubusercontent.com/99250573/203368747-9c3c45ac-af43-4ec3-844c-0d97377001e4.png)

Yours truly,
CUMGroup

**Comment by Tinf21-B4-SE-Project** [comment]
Hey there CUMGroup!

The implementation of your stats view is looking really nice! It's very easy to understand what is being shown. Good to hear that other features are also being worked on and we are very excited to take a look at them once they will be posted. Keep up the good work!

Team cybuy

**Reply by AlexMi-Ha** [reply]
Hello there Team cybuy,
Thanks for spreading positivity ^^ Always nice to get feedback from you. We all are working hard to bring you new features and demos every week 🔥

Yours truly,
CUMGroup

**Comment by vans1646** [comment]
Hi team AMOGUS,
because of the delicious looking pancakes we're getting really hungry now, thanks for that. But we are also very hungry for your future updates! 

The stats you developed are giving every info needed so good job there! We are excited for more screenshots of the login and register page as you will probably come up with another great design which fits the aesthetic of your project. 

It's really good that every team member is participating in the development and can now work with Angular. Even though every member is doing their own stuff, we have the feeling that it's coming from the same mould so you probably also communicate a lot about how it should be done. Keep that going!

Cheers,
capyclue

**Reply by jas20202** [reply]
Howdy Capyclue,

it's great to hear, that you are also hungry for our future updates! 
(Unfortunately they won't include any real pancakes as we haven't figured out how to post them in GitHub yet 😳)

Thanks for the appreciation on the stats view ^^ if you are interested in our login and register view, you can see some screenshots in our newest Blog post 😉😊

We are trying to integrate every crewmember into different tasks each week and continuously stay in touch to discuss design choices. So we are glad to hear that it works very well ^^

Anyways, have a great week, hope to see you again!
Yours truly,
CUMGroup

**Comment by MikoHeuken** [comment]
Hello Team AMOGUS,

we think the design is very pretty and you can see all the important stats in on first view.
Will there be an option to switch between Lightmode and Darkmode? And have you planned an distinction of stats between diffrent catagories?

Team Ultimate Recipe Hunter

**Reply by AlexMi-Ha** [reply]
Hi Team Ultimate Recipe Hunter,
switching themes is not on our priority list right now, but we already thought about implementing this. A distinction between categorie stats is in our mind but not the main focus now. Maybe those features will come in a later update ^^

Yours truly,
CUMGroup

**Comment by mikaWohlfart** [comment]
Hello Team Amogus,

we think the way you are going is really great. Your documents look clean and correct. It would be good if you could summarize the development and the people involved in a small table. This would help the clarity. 

Keep up the good work!

Framgang

**Reply by AlexMi-Ha** [reply]
Howdy Framgång,
thanks for taking a look over our documents. It is important to us that our documentation is as clear as possible.

BUT It seems a bit SUS ඞ to request such a well formatted table of the development authors. What statistics are you trying to collect 👀

Yours truly,
CUMGroup
